docs/tutorial.md

```diff
@@ -151,8 +151,10 @@ gate to every qubit whose row index plus column index is odd.  To
 do this we write
 ```python
 circuit = cirq.Circuit()
-circuit.append(cirq.H(q) for q in qubits if (q.row + q.col) % 2 == 0)
-circuit.append(cirq.X(q) for q in qubits if (q.row + q.col) % 2 == 1)
+circuit.append([cirq.H.on(q) for q in qubits if (q.row + q.col) % 2 == 0],
+               strategy=cirq.InsertStrategy.NEW_THEN_INLINE)
```

Comment:
I would avoid specifying the insertion strategy here; update the later text instead. It's confusing to throw this detail in in the middle of a tutorial.

Reply:
Done